session/src/main/java/org/apache/iotdb/session/SessionUtils.java

```diff
@@ -149,4 +158,32 @@ private static void getValueBufferOfDataType(
             String.format("Data type %s is not supported.", dataType));
     }
   }
+
+  public static List<EndPoint> parseSeedNodeUrls(List<String> nodeUrls) {
+    if (nodeUrls == null) {
+      return Collections.emptyList();
+    }
+    List<EndPoint> endPointsList = new ArrayList<>();
+    for (String nodeUrl : nodeUrls) {
+      EndPoint endPoint = parseNodeUrl(nodeUrl);
+      endPointsList.add(endPoint);
+    }
+    return endPointsList;
+  }
+
+  private static EndPoint parseNodeUrl(String nodeUrl) {
+    EndPoint endPoint = new EndPoint();
+    String[] split = nodeUrl.split(":");
+    if (split.length != 2) {
+      return null;
+    }
+    String ip = split[0];
+    try {
+      int rpcPort = Integer.parseInt(split[1]);
+      return endPoint.setIp(ip).setPort(rpcPort);
+    } catch (NumberFormatException e) {
+      logger.warn("parse url fail {}", nodeUrl);
+    }
+    return endPoint;
+  }
```

Comment:
If the user given one wrong URL format,  in the code implement, the `NumberFormatException` may occur, you will return one `empty` endPoint.
In my opinion, if the seed URL format error, It's better rethrow the exception to the user.